acceptance/tests/ssl/crl_retrieval_and_replacement.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Crl retrieval and replacement from master" do
+
+  tag 'risk:medium',
+      'server'
+
+  with_puppet_running_on(master, {}) do
+    agents.each do |agent|
+      ca_crl_path = on(master, puppet('config print cacrl --section master')).stdout
+      crl_path = on(agent, puppet('config print hostcrl --section agent')).stdout
+
+      step "When a newer crl is available on master" do
+        on(master, "touch #{ca_crl_path}")
```

Comment:
Is this test sensitive to clock skew between master and agents, eg if the agent's clock is ahead of the masters by 10 secs, will this touch still be in the past relative to the agent? Can we touch the path so it's an hour ahead to make it less sensitive?

Reply:
Sure... I think I can do:
one_hour_ahead = (Time.now + 3600).strftime("%Y%m%d%H%M")

on(master,  "touch -t #{one_hour_ahead} #{ca_crl_path}")